client/templates/hangout/hangout-item.js

```diff
@@ -47,17 +47,18 @@ Template.hangoutItem.helpers({
   },
   getDate: function(hangout) {
     var tz = TimezonePicker.detectedZone();
+    var startEndDiffDays = (hangout.end - hangout.start)/(1000*60*60*24)
+    var startTime = moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z')
+    // If endtime is in 24 hours (95% of 24 hours), then show "To Be Announced". Else show the end time
+    var endTime = startEndDiffDays < 0.95 ? moment(hangout.end).tz(tz).format('MMMM Do h:mm a z'): 'To Be Announced '
```

Comment:
The ternary does make that line long and awkwardly looking. I can change that. I will probably change it to something like:

`var endTime = value1`
`if (condition){`
`    endTime = value2`
`}`

Is that okay?